scripting/get5.sp

```diff
@@ -294,6 +298,8 @@ public void OnPluginStart() {
   g_TeamTimeToStartCvar = CreateConVar(
       "get5_time_to_start", "0",
       "Time (in seconds) teams have to ready up before forfeiting the match, 0=unlimited");
+  g_TechCaptainRestrict = CreateConVar("get5_captain_initiate_technical_pause", "0",
```

Comment:
I realize this is disabled by default - but I don't really see the purpose of this too much. I'd rather avoid adding tons of extra cvars and don't want to make users rely on who gets marked as "captain".